package.json

```diff
@@ -26,7 +26,6 @@
     "@types/jsonwebtoken": "^8.3.3",
     "@types/pino": "^5.8.10",
     "@types/uuid": "^3.4.5",
-    "dotenv-cli": "^2.0.1",
```

Comment:
I don't think we need this as we have no environment variables in use. If we did use it, I would consider `devDependency` as more appropriate.